app/controllers/api/v1/organization_controller.rb

```diff
@@ -36,8 +36,8 @@ def create
 
     name = params[:name]
 
-    if name.nil?
-      render stauts: 200,
+    if name.nil? || name == ""
+      render status: :forbidden,
```

Comment:
Let's change this to :bad_request. This is more convention rather than this actually being wrong but the idea is that we want to set a nice standard of communication between the client (could be any client, not just ours) and server. Often when a status of forbidden is returned, we might mean that the user does not have access to a specific api call based on their permissions. In this case, the client might have some code written that checks all statuses and if forbidden is returned it'll display a specific message vs if 404 Not Found was returned vs 400 Bad Request, etc.

So I like to return 400 Bad Request if something in the request was incorrect (i.e. passed in a null or invalid email or something).